client/packages/plugins/api/hooks/usePluginData.ts

```diff
@@ -26,7 +34,7 @@ export const usePluginData = ({ pluginCode, filter }: PluginDataProps) => {
   };
 
   const { data, isError, isLoading } = useQuery({
-    queryKey: [PLUGIN_DATA, storeId, pluginCode],
+    queryKey: [PLUGIN_DATA, storeId, pluginCode, queryKey],
```

Comment:
I realised that we needed a way to invalidate the query for individual items, so added a `queryKey` prop to this hook. I'm using it to pass in the line ID for the individual item modal view. Otherwise, the query wasn't fetching the new value when going directly to the next item with "OK and Next".